easybuild/easyconfigs/l/libheif/libheif-1.12.0-GCC-11.2.0.eb

```diff
@@ -12,14 +12,26 @@ source_urls = ['https://github.com/strukturag/libheif/releases/download/v%(versi
 sources = [SOURCE_TAR_GZ]
 checksums = ['e1ac2abb354fdc8ccdca71363ebad7503ad731c84022cf460837f0839e171718']
 
+builddependencies = [
+    ('CMake', '3.22.1'),
+]
+
 dependencies = [
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.6'),
     ('libde265', '1.0.8'),
     ('x265', '3.5'),
 ]
```

Comment:
```suggestion
    ('Gdk-Pixbuf', '2.42.6'),
]
```

Reply:
build succeeds for me when `Gdk-Pixbuf` is added

Reply:
@boegel ping?

Reply:
@verdurin fixed with 60bfb0c